src/lt/plugins/clojure.cljs

```diff
@@ -68,13 +68,15 @@
           (run-local-server (clients/client! :nrepl.client))))))
 
 (behavior ::on-eval.clj
+          :desc "Clojure: Eval editor content"
           :triggers #{:eval}
           :reaction (fn [editor]
                       (object/raise clj-lang :eval! {:origin editor
                                                      :info (assoc (@editor :info)
                                                              :print-length (object/raise-reduce editor :clojure.print-length+ nil)
                                                              :code (watches/watched-range editor nil nil nil))})))
 (behavior ::on-eval.cljs
+          :desc "Clojurescript: Eval editor content"
```

Comment:
Dunno if it's explicitly stated anywhere. Probably worth documenting in wiki/docs. I'd recommend reading more plugins and especially writing a plugin to familiarize yourself with conventions